docs/content/getting-started/create-new-project.mdx

```diff
@@ -73,12 +64,20 @@ Here's a breakdown of the files and directories that are generated:
       <td>pyproject.toml</td>
       <td>
         A file that specifies package core metadata in static, tool-agnostic
-        way. <br />
+        way.
+        <br />
+        It includes a <code>tool.dagster</code> section which references to your
+        Dagster project as a python package. This allows you to type just
```

Comment:
```
 It includes a <code>tool.dagster</code> section which references to your
 Dagster project as a python package. This allows you to type just
```

- `python` > `Python`
- Should this be something like `It includes a tool.dagster section that references your Dagster project...`?

Reply:
I think we should explicitly avoid the term "Dagster project" in these more specific contexts since despite the name of the command, we don't have a formal notion of project. 

Instead let's be specific about the behavior. "A python package with Dagster definitions defined and discoverable at the top-level"